tests/test_slogpy_console.py

```diff
@@ -0,0 +1,16 @@
+"""Import slogpy from slog"""
```

Comment:
This is not a valuable docstring for this file/module.  This is the place where you should explain the reason why you created this file, i.e. what were you thinking when you created it and why did you do it, what do you think someone should be thinking about when they look at this code.

Since there are now other unit tests that test info(), I would just delete this whole file.